### PR TITLE
Autofocusing functionality is enabled.

### PR DESCRIPTION
**Pull request by lasuax**

### CODE REVIEW

**Comment by lasuax** [comment]
@hugomedl Here it is.

**Comment by fantapop** [comment]
@hugomedl any plans to merge this one in?

File: android/src/main/java/com/rngocoder/BroadcastView.java [hunk 1]
```diff
@@ -93,9 +98,16 @@ public void onHostResume() {
             cameraView.startPreview();
         }
 
-    }
-
+        if (goCoder != null && cameraView != null) {
+            if (mAutoFocusDetector == null)
+                mAutoFocusDetector = new GestureDetectorCompat(localContext, new AutoFocusListener(localContext, cameraView));
```
Comment (hugo-ns) [inline android/src/main/java/com/rngocoder/BroadcastView.java:102]: This line fails to compile, the "AutoFocusListener" class can't be find in this PR.  I assume you're using the same from the GoCoderAndroid Sdk example, please include it.

File: android/src/main/java/com/rngocoder/BroadcastView.java [hunk 2]
```diff
@@ -55,7 +58,9 @@ public String toString() {
     private boolean muted = false;
     private int sizePreset;
     private WowzaGoCoder goCoder;
-
+    private boolean autoFocus = true;
```
Comment (hugo-ns) [inline android/src/main/java/com/rngocoder/BroadcastView.java]: This autoFocus is not being used anywhere, it should be removed.

**Review by hugo-ns — CHANGES_REQUESTED** [review]
The changes looks good, there's just missing files that it's stopping the project to compile

**Review by lasuax — COMMENTED** [review]
Added the missing class. Sorry about that.

**Review by hugo-ns — APPROVED** [review]

**Pull request merged**